balancer-js/src/modules/pools/factory/linear/linear.factory.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { parseFixed } from '@ethersproject/bignumber';
+import { AddressZero } from '@ethersproject/constants';
+import { assert } from 'chai';
+
+import { BalancerError, BalancerErrorCode } from '@/balancerErrors';
+import { BalancerSDK } from '@/modules/sdk.module';
+import { ADDRESSES } from '@/test/lib/constants';
+import { BalancerSdkConfig, Network, PoolType } from '@/types';
+
+const network = Network.GOERLI;
+const sdkConfig: BalancerSdkConfig = {
+  network,
+  rpcUrl: '',
+};
+const balancer = new BalancerSDK(sdkConfig);
+
+describe('Linear Factory - Unit tests', async () => {
+  const factory = balancer.pools.poolFactory.of(PoolType.AaveLinear);
+  context('Create', async () => {
+    const rightCreateParameters = {
```

Comment:
I was reading this "right" as being opposed to "left" and it didn't make much sense 😅 
How about `validCreateParameters` or even `validParams` if you'd like to make it shorter? 🙃